neuraxle/base.py

```diff
@@ -2055,6 +2056,27 @@ def load_step(self, step: 'TruncableSteps', context: ExecutionContext) -> 'Trunc
         return step
 
 
+def join_apply_results(results, new_results) -> Union[Dict, Iterable]:
```

Comment:
I would delete this method.

Reply:
why ? so people would have to code it themselves ?